compiler/erlang_check.erl

```diff
@@ -245,29 +244,26 @@ check_module(File) ->
             report,
             % By adding debug_info, we ensure that the output of xref:m will
             % contain the caller MFAs too.
-            debug_info,
-            {i, AbsDir ++ "/include"},
-            {i, AbsDir ++ "/../include"},
-            {i, AbsDir ++ "/../../include"},
-            {i, AbsDir ++ "/../../../include"}],
```

Comment:
Heck, I can't remember the reason I merged it - it does look fishy indeed. @hcs42 we could tag the current release, and bump it after this gets merged? Most of vim plugin managers support pinning tags I suppose?